package.json

```diff
@@ -91,21 +91,23 @@
     "ts-node": "^10.2.1",
     "typedoc": "^0.24.0",
     "typescript": "^4.6.4",
-    "webpack": "^5.88.2",
+    "webpack": "^5.91.0",
     "webpack-assets-manifest": "^5.1.0",
-    "webpack-bundle-analyzer": "^4.9.1",
+    "webpack-bundle-analyzer": "^4.10.1",
     "webpack-cli": "^5.1.4",
-    "webpack-dev-server": "^4.15.1",
-    "webpack-merge": "^5.9.0",
+    "webpack-dev-server": "^5.0.4",
+    "webpack-merge": "^5.10.0",
     "webpack-subresource-integrity": "^5.1.0",
     "yargs": "^17.7.2"
   },
   "pnpm": {
     "overrides": {
       "axios": "^1.6.0",
       "dns-packet": "^1.3.2",
-      "follow-redirects": "^1.15.4",
+      "express": "^4.19.2",
+      "follow-redirects": "^1.15.6",
```

Comment:
Can you please add explanations for these overrides into the overrides-explanations section below this? Particularly the details about why we need the overrides and when we know it will be safe to remove them.

Thanks!